src/docs/spec-formats-rdf.md

```diff
@@ -0,0 +1,429 @@
+# The SSSOM/RDF serialisation format
+
+This section defines how to represent a SSSOM mapping set as a
+[RDF model](https://www.w3.org/TR/rdf11-concepts/).
+
+## RDF formats
+
+The RDF model that represents a SSSOM mapping set is independent of the concrete
+format that may be used to serialise the model.
+
+It is RECOMMENDED that implementations support reading and writing a SSSOM set
+from and to the [RDF Turtle](https://www.w3.org/TR/turtle/) format at least.
+They MAY support any other RDF concrete format (e.g. RDF/XML, TriG, N-Triples,
+etc.).
+
+This specification does not mandate how a concrete RDF syntax is to be used. For
+example, if the RDF syntax allows named resources and predicates to be
+serialised as either IRIs or CURIEs, if is left at the discretion of the
+implementations (or their users) to decide which form to use.
+
+<a id="sssom-slots"></a>
+
+## Representation of slots
+
+A metadata slot on any given SSSOM object (such as a `Mapping` or a
+`MappingSet`) MUST be represented as a RDF triple where:
+
+- the subject is the resource representing the SSSOM object;
+- the predicate is either:
+  - the property indicated by the `URI` field in the LinkML description of the
+    slot, if such a field is present;
+  - or a property constructed by concatenating the `https://w3id.org/sssom/`
+    namespace and the name of the slot;
+- the object is the value of the slot.
+
+### Representation of slot values
+
+The following rules determine how the value of a slot is represented as the
+object of a RDF triple.
+
+#### For slots typed as `sssom:EntityReference`
+
+(e.g. `subject_id`, `mapping_justification`, `subject_source`…)
+
+The value MUST be represented as a named RDF resource (IRI).
+
+#### For slots typed as `sssom:NonRelativeURI`
+
+(e.g. `license`, `mapping_provider`, `issue_tracker`…)
+
+The value MUST be represented as a named RDF resource (IRI).
+
+#### For slots typed as `linkml:date`
+
+(e.g. `mapping_date`, `publication_date`)
+
+The value MUST be represented as a `xsd:date` literal.
+
+#### For slots typed as `linkml:double`
+
+(e.g. `mapping_set_confidence`, `confidence`, `similarity_score`)
+
+The value MUST be represented as a `xsd:double` literal.
+
+#### For slots typed as an enumeration
+
+(e.g. `sssom_version`, `mapping_cardinality`, `subject_type`…)
+
+If the permissible values for the enumeration are defined in the LinkML model as
+having an associated `meaning` property, then the value MUST be represented as a
+named RDF resource with the indicated property. Otherwise, the value MUST be
+represented as a `xsd:string` literal.
+
+#### For slots typed as a SSSOM object
+
+(e.g. `mappings`, `extension_definitions`)
+
+The value MUST be represented as a RDF resource. Whether the resource is named
+(IRI) or not (blank node) will depend on the type of the object, see the
+[section on representing SSSOM objects](#sssom-objects) below for details.
```

Comment:
this is a bit confusing why it's documented here but not in full, can you make an explicit enumeration here of the three things this applies to an an link to each?

```suggestion
The value MUST be represented as a RDF resource. Whether the resource is named
(IRI) or not (blank node) will depend on the type of the object, see the following sections:

1. [mappings](#representation-of-a-mapping-object)
2. [mapping set)(#representation-of-a-mappingset-object)
3. [extension definition](#representation-of-a-extensiondefinition-object)
```